docs/howto/openstack/neutron/del-network.md

```diff
@@ -0,0 +1,553 @@
+    ```
+        
+    The absence of any output means the command was successful. Take a look 
+    yourself:
+        
+    ```bash
+    openstack network list --internal
+    ```
+    ```plain
+    +--------------------------------------+--------------+--------------------------------------+
+    | ID                                   | Name         | Subnets                              |
+    +--------------------------------------+--------------+--------------------------------------+
+    | 1f94d315-7ca1-4d44-acc1-09c6c650df74 | mayo         |                                      |
+    | e0c4ce17-2722-4777-8140-d6c87479e190 | network-kna1 | 421d8fd2-dd7f-4f7c-9a51-42ef4a866dd9 |
+    +--------------------------------------+--------------+--------------------------------------+
+    ```
+    
+## Networks with no subnet and no router
+
+You may directly, without the slightest preparation, delete networks 
+like these. For our demonstration, we created a network named `mayo`, 
+with no subnet and no router in front of it.
+
+=== "{{gui}}"
+    While viewing all available networks, click the red three-dot icon at 
+    the right-hand side of the `mayo` row and select _Delete Network_. You 
+    will have to confirm the action, and the network will be gone as soon 
+    as you do.
+    
+    ![Delete Mayo](assets/del-net/shot-17.png)
+=== "OpenStack CLI"
+    Once more, take a look at all remaining networks:
+        
+    ```bash
+    openstack network list --internal
+    ```
+    ```plain
+    +--------------------------------------+--------------+--------------------------------------+
+    | ID                                   | Name         | Subnets                              |
+    +--------------------------------------+--------------+--------------------------------------+
+    | 1f94d315-7ca1-4d44-acc1-09c6c650df74 | mayo         |                                      |
+    | e0c4ce17-2722-4777-8140-d6c87479e190 | network-kna1 | 421d8fd2-dd7f-4f7c-9a51-42ef4a866dd9 |
+    +--------------------------------------+--------------+--------------------------------------+
+    ```
+        
+    Since `mayo` has no subnet, issue a single command to delete it:
+        
+    ```bash
+    openstack network delete mayo
+    ```
+        
+    And, yes, it is still a good idea to check yourself:
+        
+    ```bash
+    openstack network list --internal
+    ```
+    ```plain
+    +--------------------------------------+--------------+--------------------------------------+
+    | ID                                   | Name         | Subnets                              |
+    +--------------------------------------+--------------+--------------------------------------+
+    | e0c4ce17-2722-4777-8140-d6c87479e190 | network-kna1 | 421d8fd2-dd7f-4f7c-9a51-42ef4a866dd9 |
+    +--------------------------------------+--------------+--------------------------------------+
+    ```
+
+## Recap: Of networks and towns
```

Comment:
Use just "Recap" or "Summary" here.

Reply:
Hm, yes, why not, but please read about my rationale below.